src/components/Signup/Signup.tsx

```diff
@@ -97,10 +97,10 @@ const Signup: React.FC<SignupProps> = ({ isShow, returnShow }) => {
     <>
       <Modal show={show} onHide={handleClose}>
         <Modal.Header closeButton>
-          <Modal.Title>Sign Up</Modal.Title>
+          <Modal.Title>Log In</Modal.Title>
         </Modal.Header>
         <Modal.Body className="d-flex justify-content-center align-items-center m-5">
-          <GoogleButton type="light" onClick={logGoogleUser} />
+          <GoogleButton type="light" onClick={logGoogleUser} label="Login with Google"/>
```

Comment:
@RamakrushnaBiswal Will the user always sign up and never log in ?
You just had to add this `label` prop. 
Make this `Continue with Google`

Reply:
Okay

Reply:
completed  [833f61e](https://github.com/Tanay-ErrorCode/lupo-skill/pull/146/commits/833f61e1f5396f23b4aa4fd4f8078698688b5d13)